lib/twingly/http.rb

```diff
@@ -13,6 +13,7 @@ module Twingly
   module HTTP
     class ConnectionError < StandardError; end
     class UrlSizeLimitExceededError < StandardError; end
+    class RedirectLimitReached < StandardError; end
```

Comment:
```suggestion
    class RedirectLimitReachedError < StandardError; end
```

That would make it match the name of the other error defined above.

Reply:
Sure

Reply:
b1137e3